curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-fetch-promises-and-async-await-by-building-an-fcc-forum-leaderboard/64388945b59341057e955fbb.md

```diff
@@ -0,0 +1,162 @@
+---
+id: 64388945b59341057e955fbb
+title: Step 19
+challengeType: 0
+dashedName: step-19
+---
+
+# --description--
+
+In this project, you will be using the freeCodeCamp forum API. Requesting data from an API is an asynchronous operation, so you will need to use an asynchronous function with the `async` keyword.
+
+Here is an example of an asynchronous function.
+
+```js
+async function example() {
+  console.log("this is an example");
+}
+```
```

Comment:
As we are providing the HTML and CSS for this advanced project, we can remove the steps from 1 to 18(including 18).

Here are the updated instructions for the first step:

In this project, you will be building a clone of the fcc leaderboard. The HTML and CSS have been provided for you. Feel free to explore them.

You will also be using the freeCodeCamp forum API. Requesting data from an API is an asynchronous operation, so you will need to use an asynchronous function with the `async` keyword.

Here is an example of an asynchronous function.

```js
async function example() {
  console.log("this is an example");
}
```

When you are ready, create an async function named `fetchData`.